keps/sig-node/281-dynamic-kubelet-configuration/README.md

```diff
@@ -30,157 +31,209 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [X] (R) Design details are appropriately documented
 - [X] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
 - [X] (R) Graduation criteria is in place
-- [ ] (R) Production readiness review completed
+- [X] (R) Production readiness review completed
 - [ ] Production readiness review approved
 - [ ] "Implementation History" section is up-to-date for milestone
 - [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
 - [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
 
 ## Summary
 
-Dynamic Kubelet Configuration allows a new Kubelet configurations to be rolled out in a live cluster.
+Dynamic Kubelet Configuration allows a new Kubelet configurations to be rolled
+out in a live cluster.
 
 The feature predates the KEP process as it is defined today. Please find
 motivation, goals, and design section in [community repository](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/dynamic-kubelet-configuration.md).
 
+Dynamic Kubelet Configuration feature is deprecated at the version v1.22 due to
+lack of interest in promoting it to stable. See [Deprecation](#deprecation).
+
+## Deprecation
+
+Dynamic Kubelet Configuration feature was promoted to beta in [v1.11, 3 July 2018](https://kubernetes.io/blog/2018/06/27/kubernetes-1.11-release-announcement).
+As per the [Avoiding Permanent Beta](https://kubernetes.io/blog/2020/08/21/moving-forward-from-beta/#avoiding-permanent-beta)
+policy this feature is marked for deprecation as there were no interest expressed to
+promote the feature to stable. Removal of Dynamic Kubelet Configuration logic will
+simplify code and improve code reliability.
+
 ### Risks and Mitigations
 
-TODO for the feature deprecation effort
+The deprecation announcement may uncover wide feature usage.
+So far only one complain [was expressed](https://github.com/kubernetes/enhancements/issues/281#issuecomment-695751859).
```

Comment:
A note here: We heard this complain in the past and suggested to help promoting this to beta, but failed moving forward. There are alternatives, but carrying on this alpha feature is overhead for the community. Sergey, thanks for digging into the history here.

Reply:
s/beta/stable above.